RELEASE.md

```diff
@@ -7,32 +9,52 @@
 ### Notes
 
 - The `master` branch must refer to frontier `paritytech/frontier/master` or
-  `purestake/frontier/vX.X-hotfixes`
-
-### Pre-Release Checklist
-
-- [ ] Purge requirement has been communicated and scheduled with ops
-- [ ] Purge announcement has been prepared and scheduled
-- [ ] Documentation/Website/Tutorials have been updated
-- [ ] Runtime version has been updated
-- [ ] Tests are passing on StageNet
-- [ ] Ensure Type changes are reflected into PolkadotJs
+  `purestake/frontier/moonbeam-polkadot-vX.Y.Z`
 
 ### Release Workflow
 
+To release `vX.Y.Z`:
+
+1. Create a PR with increment client version
+1. Get that PR approved and merged
+1. Tag master with `vX.Y.Z` and push to github
+1. Start the github action "Publish Binary Draft" (on master branch)
+1. Review the generated Draft and clean a bit the messages if needed (keep it draft)
+1. Test the new client on internal tests networks (stagenet/moonsama/moonlama)
```

Comment:
This is the important part, I think we need this to have its own "preflight checklist" somewhere. That might include making sure certain things are working as expected:

- sudo
- democracy
- call filter
- precompiles

etc. etc.

Reply:
(I'm thinking more for runtime releases than client releases...)

Reply:
Other ideas: ensure migrations are up to date and add any comments to them (e.g. comment out which ones have already run, like we do here: https://github.com/PureStake/moonbeam/pull/1180)